types/git-url-parse/git-url-parse-tests.ts

```diff
@@ -3,6 +3,11 @@ import gitUrlParse = require("git-url-parse");
 gitUrlParse("git@github.com:IonicaBizau/node-git-url-parse.git"); // $ExpectType GitUrl
 gitUrlParse("git@github.com:IonicaBizau/node-git-url-parse.git").toString(); // $ExpectType string
 gitUrlParse("git@github.com:IonicaBizau/node-git-url-parse.git").toString("git+ssh"); // $ExpectType string
+gitUrlParse("git@github.com:IonicaBizau/node-git-url-parse.git").port; // $ExpectType string
```

Comment:
This seems odd when the package source documents `number`...

Reply:
yes, but the `git-url-parse` package isn't changing it. it was a number long ago in older version.
Since some recent major it's using another base package which uses string.
I've opened PR's to the repos to fix the docs issue.

Reply:
I mean it's coming from `git-url-parse>git-up>parse-url>parse-path`.

And `parse-path` is using `new U

Reply:
- https://github.com/IonicaBizau/git-url-parse/pull/177